mllib/src/main/scala/org/apache/spark/ml/regression/LinearRegression.scala

```diff
@@ -438,7 +438,6 @@ class LinearRegressionModel private[ml] (
     }
   }
 
-
```

Comment:
For me it's just below the level of changing on its own. It has no functional impact, and doesn't really have any impact on readability. It's the kind of thing I'd only change if touching the surrounding code anyway.